target/snitch_cluster/sw/snax/gemm/src/snax-gemm-lib.c

```diff
@@ -23,15 +34,17 @@ void base_gemm(uint8_t m, uint8_t k, uint8_t n, int8_t* A, int8_t* B,
                 C_cpu[i * n + j] = 0;
             }
             for (int s = 0; s < k; s++) {
-                C_cpu[i * n + j] = C_cpu[i * n + j] + (int32_t)A[i * k + s] *
-                                                          (int32_t)B[s + j * k];
+                C_cpu[i * n + j] = C_cpu[i * n + j] +
+                                   ((int32_t)A[i * k + s] - subtraction_a) *
+                                       ((int32_t)B[s + j * k] - subtraction_b);
```

Comment:
Would this make sense @xiaoling-yi ?
```suggestion
                                   ((int32_t)A[i * k + s] - (int32_t)subtraction_a) *
                                       ((int32_t)B[s + j * k] - (int32_t)subtraction_b);
```

Reply:
Applied!